### PR TITLE
fix doc example so that it works

### PR DESCRIPTION
**Pull request by spencerbart**

### CODE REVIEW

**Comment by spencerbart** [comment]
I don't know if we should put a message saying that the sha2 feature needs to be enabled to get this example to work.

File: src/lib.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@
 //!
 //! ## OAEP encryption
 //! ```
-//! use rsa::{RsaPrivateKey, RsaPublicKey, Oaep};
+//! use rsa::{RsaPrivateKey, RsaPublicKey, Oaep, sha2};
```
Comment (tarcieri) [inline src/lib.rs:22]:
```suggestion
//! use rsa::{RsaPrivateKey, RsaPublicKey, Oaep, sha2::Sha256};
```

**Comment by tarcieri** [comment]
> message saying that the sha2 feature needs to be enabled

sure

**Comment by tarcieri** [comment]
Closing in favor of #381

**Pull request closed without merging**